.travis.yml

```diff
@@ -1,7 +1,18 @@
 language: c
-install: wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-opam.sh
-script: bash -ex .travis-opam.sh
-dist: trusty
+sudo: false
+services:
+  - docker
+install: wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-docker.sh
+script: bash ./.travis-docker.sh
 env:
-  - PACKAGE="xenctrl" OCAML_VERSION=4.01
-  - PACKAGE="xenctrl" OCAML_VERSION=latest
+ global:
+   - PACKAGE="xenctrl"
+ matrix:
+   - DISTRO=debian-stable OCAML_VERSION=4.01.0
+   - DISTRO=debian-testing OCAML_VERSION=4.02.3
+   - DISTRO=debian-unstable OCAML_VERSION=4.03.0
+   - DISTRO=ubuntu-12.04 OCAML_VERSION=4.02.3
```

Comment:
I've pushed a patch to your branch commenting out the bad distros. I thought it would be nice to comment out the lines rather than delete them as a visible reminder that something doesn't work (like a negative test case). Hopefully I've got the travis comment syntax right :/